package.json

```diff
@@ -174,14 +175,14 @@
     "react-native-reanimated": "^2.9.1",
     "react-native-render-html": "^6.3.1",
     "react-native-responsive-screen": "^1.4.1",
-    "react-native-safe-area-context": "^3.3.2",
+    "react-native-safe-area-context": "^4.9.0",
     "react-native-screen-brightness": "^2.0.0-alpha",
-    "react-native-screens": "^2.18.1",
+    "react-native-screens": "^3.29.0",
```

Comment:
@LazyAfternoons did you forgot to push the commit that revert this?

Reply:
Sorry, the push failed and I didn't notice!